Sources/WordPressKit/Services/AccountServiceRemoteREST+SocialService.swift

```diff
@@ -24,7 +24,7 @@ extension AccountServiceRemoteREST {
                                        oAuthClientSecret: String,
                                        success: @escaping (() -> Void),
                                        failure: @escaping ((Error) -> Void)) {
-        let path = self.path(forEndpoint: "me/social-login/connect", withVersion: ._1_1)
+        let path = self.path(forEndpoint: "me/social-login/connect", with: ._1_1)
```

Comment:
The compiler changed this, which was a bit annoying because it made the diff quite noisy.

I guess the compiler is happy with just "with" when the type on the other side is "strong", i.e. a `typedef NS_ENUM` that converts neatly to `enum` in Swift.

Reply:
You can avoid this name change by adding a `NS_SWIFT_NAME(path(forEndpoint:withVersion:))` to the function in the ObjC header.

Reply:
Nice! Much better diff, thanks for the tip @crazytonyli 

I think the Swift API is better this way, too. `with: ._1_1` was not as clear as `withVersion: ._1_1`.